### DISCUSSION TITLE
I can't get the plugin to work

### DISCUSSION THREAD

**Discussion by IamTheCanary** [natefrisch01/Graph-Link-Types#33] [General]
I'm trying to install and configure the Graph Link Types plugin to enhance my graph view in Obsidian, but I'm having some trouble getting it to work properly. I've already installed Dataview, but I'm not sure how to structure the metadata correctly for the link types.

Nothing seems to change, and I can't figure out how to use it properly. Here's a screenshot.

![test](https://github.com/natefrisch01/Graph-Link-Types/assets/134302150/14fbafb3-94db-4c8a-9274-27c21c740991)

What's more, there's a whole panel that I can't find on my version, I tried to see how to configure the plugin but there were only 3 switches.

![Capture d’écran 2024-05-12 053339](https://github.com/natefrisch01/Graph-Link-Types/assets/134302150/4b9d25da-18dc-4a10-8f27-4bb6ad2b7503)

I don’t understand how others manage to have such configurations when I’m just struggling to establish a link between two notes.

**Comment by LynetteCullens** [comment]
I can't find it either.

**Comment by LynetteCullens** [comment]
Figured out that all link properties have to be at the top. It has to be the first field/fields on your file to register. This was supposed to be fixed, but it's not.

Edit: Panel can only be seen on local graph view.

**Reply by natefrisch01** [reply]
This is still an open issue (#25).
